airtable/plugin.py

```diff
@@ -83,21 +113,22 @@ def _resolve_error_message(
 
 @dataclass
 class AirtableRecord:
-    id: str
+    airtable_record_id: AirtableRecordID
```

Comment:
Is it necessary/helpful to prefix the field name with `airtable_`?  Maybe @jngiam also has insight here?  I would think that either `id` + the type hint would be sufficient and avoid the stutter.

Reply:
no need to prefix field names.

we want to prefix (a) dataclasses and (b) actions names.

[though for action names, eventually we want module style naming i think (airtable.get....)]

Reply:
(i wanted to suggest using "Id" instead of "ID" for consistency, since it's a shorthand for "Identifier", the D should be lower case... but i just realize that this is currently bad in google.py where we use "ID" instead of "Id" :( )

we can file an issue to fix this later. 

naming is really critical for code gen, so being consistent everywhere helps a lot.